sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs

```diff
@@ -386,11 +386,10 @@ public DebugStore (string[] loaded_files)
 			var asm_files = new List<string> ();
 			var pdb_files = new List<string> ();
 			foreach (var f in loaded_files) {
-				var file_name = f.ToLower ();
-				if (file_name.EndsWith (".pdb", StringComparison.Ordinal))
-					pdb_files.Add (file_name);
+				if (f.EndsWith (".pdb", StringComparison.OrdinalIgnoreCase))
+					pdb_files.Add (f);
 				else
-					asm_files.Add (file_name);
+					asm_files.Add (f);
```

Comment:
previously this was lowercasing everything in pdb_files and asm_files. You're removing that now and the list will retain loaded_files' case. Is that intentional?

Reply:
yup